### PR TITLE
Fix Execute Selected not working for dirty queries

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description
The Execute Selected was not working when the query wasn't in a "dirty state".

Pending:
- [x] Test on Netlify (and make gifs)
- [x] ~Cypress test(?)~ Not that easy to select a text inside Ace Editor in Cypress :confused:

## Related Tickets & Documents
--

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
**Before**
execute-selected

**After**
execute-selected-fix

### CODE REVIEW

File: client/app/pages/queries/view.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ function QueryViewCtrl(
     }
 
     $scope.showLog = false;
-    if ($scope.isDirty) {
+    if ($scope.isDirty || !isEmpty(selectedQueryText)) {
```
Comment (ranbena) [inline client/app/pages/queries/view.js]: Why is `isDirty` being checked here actually?

Comment (arikfr) [reply client/app/pages/queries/view.js]: If the query text wasn't changed we use the results API that takes just the query id (the safe API which requires only read access). If the text was changed, we use the results API that takes SQL (requires full access).

**Comment by ranbena** [comment]
@gabrieldutra maybe this could help with ace/cypress https://github.com/cypress-io/cypress/issues/1818#issuecomment-500792891

**Comment by gabrieldutra** [comment]
@ranbena thanks, that helps with Ace Editor usages in code (avoids having to to use `{ force: true }`), but the bigger issue is to select the text in Ace. I think I'll give a last try with mouse events :+1:

**Comment by ranbena** [comment]
@gabrieldutra hmm right. Not sure it's worth the effort.

**Pull request merged**